app/registries/foreman/webpack_assets.rb

```diff
@@ -1,7 +1,18 @@
 module Foreman
   module WebpackAssets
-    def bundle_name(plugin_name)
-      plugin_name.to_s.gsub(/core$/, '').gsub(/-|_|#{remove_bundle_name_plugins}/,'')
+    # Create name for an entry
+    # E.g.
+    #   index.js -> katello.js
+    #   common_index.js -> katello_common.js
+    def bundle_name(plugin_name, entry_path = '')
+      index_prefix = File.basename(entry_path).gsub(/[_]?index[.]js$/, '')
+      bundle = plugin_name.to_s.gsub(/core$/, '').gsub(/-|_|#{remove_bundle_name_plugins}/,'')
+
+      if index_prefix == ''
```

Comment:
nitpick: .blank?

Reply:
We're using this helper also in `plugin_webpack_directories.rb` where we don't have active support included.